### ISSUE TITLE
Make it possible to move textbox after placing

### ISSUE THREAD

**Issue by oaklinq** [PintaProject/Pinta#633]
**Description**
Currently if you place a Textbox you can't move it. It would make things a lot easier if you could move it around freely before you finshed writing the text.

**Comment by cameronwhite** [comment]
I think right click + drag will move the active text around, but this could be improved to be more discoverable (e.g. just left click + dragging around the border of the text)

**Comment by oaklinq** [comment]
Thanks for the answer. Right click + Drag doesn't work for me. Am I doing something wrong?
[Screen Recording](https://github.com/PintaProject/Pinta/assets/46486944/2392f19f-a1e8-43fb-98cd-c0462bcd28e0)

**Comment by cameronwhite** [comment]
Here is it working for me - I'm on a laptop though so this is happening via two finger click and drag on the touchpad, so I'm not sure if anything's different there

https://github.com/PintaProject/Pinta/assets/303951/3b227ef4-63cc-4de3-b211-c8c47ad73287

**Comment by oaklinq** [comment]
Ah ok so the two finger click changes the mouse icon for me, but I can't move it around. 
Maybe that's a system because two finger click + move is already used for scrolling? I'm on Fedora 39 with GNOME 45.

![Bildschirmfoto vom 2023-12-27 07-36-37](https://github.com/PintaProject/Pinta/assets/46486944/5672a1de-7c78-4b5b-aab5-de13be931a77)

**Comment by cameronwhite** [comment]
yeah you might be right about your system intercepting the two finger click + drag. I don't think Pinta is responsible for switching that cursor to the "move" icon there - it should actually be switching to a "grab" / hand cursor, but in my video above that wasn't happening which is probably a separate issue

**Comment by iksi4prs** [comment]
I'm trying to switch from Paint.Net to Pinta, and I to figure out how it works.
Right click + drag works for me.
But on Paint.Net it's more friendly, as the "cross" cursor appears all the time, and not need for right-click for it,
so its obvious how to move the text.
Is current behavior final or subject to changes ?

**Comment by cameronwhite** [comment]
I think the current behaviour should be changed to support left click + drag and show a cursor to display when that can happen
